circle.yml

```diff
@@ -1990,6 +2000,7 @@ jobs:
         type: string
         default: medium+
     steps:
+      - restore_cached_workspace
```

Comment:
Do you know if this step left out of these 3 jobs intentionally @flotwig?

Reply:
No, I'm not sure.

Reply:
I'm going to keep these in then. They speed up these jobs since they don't have to do `yarn install` again